build-config/make/mtdutils.make

```diff
@@ -24,7 +24,7 @@ MTDUTILS_STAMP		= $(MTDUTILS_SOURCE_STAMP) \
 			  $(MTDUTILS_BUILD_STAMP) \
 			  $(MTDUTILS_INSTALL_STAMP)
 
-MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo
+MTDBINS = mkfs.jffs2 mkfs.ubifs ubinize ubiformat ubinfo mtdinfo ubiattach ubimkvol ubidetach ubirmvol
```

Comment:
Actually - my apologies but turns out I was wrong. I was looking at the contents of sysroot in the build directory and there busybox is being used but the final sysroot has only the mtdtools version. I didn't realise until I installed the image on switch.

What I propose is I'll modify the script so only if the binary is not present will it install the mtdtools version. That should mean that if a platform disables the busybox ubi tools they will get the mtdtools version, and otherwise they will just get the default busybox versions. That should maintain backwards compatibility but let folk like us that want the full versions have them.